style/app.scss

```diff
@@ -233,10 +233,51 @@ a:active
   color           : #4c004c;
 }
 
-#disclaimerDiv
+.popup-banner
 {
-  padding   : 0px 1em 0px 1em;
-  font-size : 11pt;
+  font-size        : 11pt;
+  position         : absolute;
+  background-color : #ffdd61;
+  padding          : 5px 15px 5px 15px;
+  z-index          : 1;
+}
+
+.popup-banner.dismiss
+{
+  transition : visibility 0s 0.5s, opacity 0.5s linear;
+  visibility : hidden;
+  opacity    : 0;
+}
+
+.popup-banner a
+{
+  color : #af16af;
```

Comment:
Don't set a different colour here. Instead, use `text-decoration: underline` (in fact, we should have `text-decoration: underline` active by default for all `a` tags, but that's a different story).